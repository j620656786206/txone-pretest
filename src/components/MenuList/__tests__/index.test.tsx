import { render } from '@testing-library/react'
import { TonicProvider, Menu } from '@tonic-ui/react'

import CustomMenuList from '../'

describe('CustomMenuList Test', () => {
  it('with dark mode, should render successfully', () => {
    const { baseElement } = render(
      <TonicProvider
        colorMode={{
          defaultValue: 'dark', // One of: 'dark', 'light'
        }}
        useCSSBaseline={true} // If `true`, apply CSS reset and base styles
      >
        <Menu>
          <CustomMenuList />
        </Menu>
      </TonicProvider>
    )

    expect(baseElement).toBeTruthy()
  })

  it('with dark mode, should match snapshot', () => {
    const { baseElement } = render(
      <TonicProvider
        colorMode={{
          defaultValue: 'dark', // One of: 'dark', 'light'
        }}
        useCSSBaseline={true} // If `true`, apply CSS reset and base styles
      >
        <Menu>
          <CustomMenuList />
        </Menu>
      </TonicProvider>
    )

    expect(baseElement).toMatchSnapshot()
  })
})
