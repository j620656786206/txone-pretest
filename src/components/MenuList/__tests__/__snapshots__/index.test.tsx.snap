// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CustomMenuList Test with dark mode, should match snapshot 1`] = `
<body>
  <div>
    <div
      class="css-1y2pe6x-Box ebasi3p0"
    />
  </div>
</body>
`;
